README.md

```diff
@@ -9,8 +9,11 @@ The firmware is based on vanilla [OpenWrt](https://openwrt.org/start) with some
 broken stuff in OpenWrt itself or for example LuCI) and additional default packages/configuration settings.
 New features like a new network concept will be part of future releases.
 
-## Release Note 1.0.1 "Hedy" - 2018-05-29
-* just a small maintenance release for Hedy-1.0.0, which brings the missing upgrade possibility from previous releases
+## Release Note 1.0.2 "Hedy" - 2019-01-xx
+* a maintenance release for Hedy-1.0.x-series
+* brings the the new feature of changing the uplink preset-type
+* a new uplink-preset "tunnelberlin-tunneldigger"
```

Comment:
Add freifunk-berlin-bbbdigger to connect to the bbb-vpn.

Reply:
also for the release-notes

Reply:
I think it would make sense to have it in the readme.  We now have two addons, bbbdigger and collectd-snmp and they both should be in the readme.  

I agree with you. the new addon should go in the release notes as well.

Reply:
please check PR #655 .
I suggest to strip down the Readme to information relevant to developers. So we can list even such packages in the Wiki and don't have to keep such information in 2 places.

I don't like the idea of listing such packages in the Readme.

Reply:
Sounds good.